Shared/Samples/Augment reality to navigate route/README.md

```diff
@@ -51,15 +51,15 @@ This sample uses Esri's [world elevation service](https://elevation3d.arcgis.com
 
 ## Additional information
 
-This sample requires a device that is compatible with ARKit 1 on iOS.
+This sample requires a device that is compatible with ARKit on iOS.
 
 Unlike other scene samples, there's no need for a basemap while navigating, because context is provided by the camera feed showing the real environment. The base surface's opacity is set to zero to prevent it from interfering with the AR experience.
 
 A digital elevation model is used to ensure that the displayed route is positioned appropriately relative to the terrain of the route. If you don't want to display the route line floating, you could show the line draped on the surface instead.
 
-**World-scale AR** is one of three main patterns for working with geographic information in augmented reality. Augmented reality is made possible with the ArcGIS Runtime Toolkit. See [Augmented reality](https://developers.arcgis.com/ios/scenes-3d/display-scenes-in-augmented-reality/) in the guide for more information about augmented reality and adding it to your app.
+**World-scale AR** is one of three main patterns for working with geographic information in augmented reality. Augmented reality is made possible with the ArcGIS Maps SDK Toolkit. See [Augmented reality](https://developers.arcgis.com/swift/scenes-3d/display-scenes-in-augmented-reality/) in the guide for more information about augmented reality and adding it to your app.
 
-Because most navigation scenarios involve traveling beyond the accurate range for ARKit/ARCore positioning, this sample relies on **continuous location updates** from the location data source. Because the origin camera is constantly being reset by the location data source, the sample doesn't allow the user to pan to calibrate or adjust the altitude with a slider. The location data source doesn't provide a heading, so it isn't overwritten when the location refreshes.
+Because most navigation scenarios involve traveling beyond the accurate range for ARKit positioning, this sample relies on **continuous location updates** from the location data source. Because the origin camera is constantly being reset by the location data source, the sample doesn't allow the user to pan to calibrate or adjust the altitude with a slider. The location data source doesn't provide a heading, so it isn't overwritten when the location refreshes.
```

Comment:
Im struggling with this paragraph. Feel free to ignore this suggestion but how about something like this?

In most navigation scenarios, this sample does not allow the user to pan to calibrate or adjust the altitude with a slider. This is because the origin of the camera is constantly being reset by the location data source. The location data source is continuously updated when the navigation is travelling beyond the accurate range for ARKit positioning.

Reply:
Welp, I'll leave this suggestion to a separate PR. 🫤

I agree it doesn't read well, but you are gonna be overwhelmed by some of the poorly phrased paragraphs if you start digging into the sample design doc rabbit hole… 🕳️🕷️🐛🕷️🪱🕷️🐞